### PR TITLE
Add methods for business hierarchy operations

### PR DESCRIPTION
**Pull request by chenhunghan**
Mainly for endpoints added in https://github.com/lensapp/lenscloud/pull/2572

### CODE REVIEW

File: src/BusinessService.ts [hunk 1]
```diff
@@ -629,6 +666,163 @@ class BusinessService extends Base {
...
+  /**
+   * List all children LBIDs of a LBID.
+   *
+   * @remarks user has to be the administrator of the business.
+   */
+  async getChildren(id: Business["id"]): Promise<Business[]> {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/businesses/${id}/businesses`;
+    const json = await throwExpected(async () => fetch.get(url), {
+      403: (error) => new ForbiddenException(error?.body?.message),
```
Comment (DmitriyNoa) [inline src/BusinessService.ts:679]: Nit. Do we need to add 401 here as well?

File: src/BusinessService.ts [hunk 2]
```diff
@@ -629,6 +666,163 @@ class BusinessService extends Base {
...
+   * Remove all children LBIDs of a LBID.
+   *
+   * @remarks user has to be the administrator of the business.
+   */
+  async removeAllChildren(id: Business["id"]) {
+    const { apiEndpointAddress, fetch } = this.lensPlatformClient;
+    const url = `${apiEndpointAddress}/businesses/${id}/businesses`;
+
+    await throwExpected(async () => fetch.delete(url), {
+      403: (error) => new ForbiddenException(error?.body?.message),
```
Comment (DmitriyNoa) [inline src/BusinessService.ts:696]: Also 401?

File: src/BusinessService.ts (hunk 1 above)
Comment (chenhunghan) [reply src/BusinessService.ts:679]: This is from NestJS guard, default to 403

File: src/BusinessService.ts [hunk 3]
```diff
@@ -629,6 +666,163 @@ class BusinessService extends Base {
...
+    const json = await throwExpected(async () => fetch.get(url), {
+      403: (error) => new ForbiddenException(error?.body?.message),
+      404: (error) => new NotFoundException(error?.body?.message),
+    });
+
+    return json as unknown as Business;
+  }
+
+  /**
+   * Remove the parent LBID of a LBID.
```
[the remark this answers is not in the document]
Comment (chenhunghan) [reply src/BusinessService.ts:737]: Yes, it's just set parentId to null on Recurly, not removing BusinessEntity in db.

File: src/BusinessService.ts (hunk 2 above)
Comment (chenhunghan) [reply src/BusinessService.ts:696]: https://github.com/lensapp/lens-platform-sdk/pull/429#discussion_r1087905662

**Review by DmitriyNoa — APPROVED** [review]

**Pull request merged**